script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -756,7 +756,7 @@ Don't ever look at things thoughtlessly, okay?"
 -- Page 232, Offset 4604.
 -- @g　室内に入った瞬間、かちん、と撃鉄の落ちる音を聞いた。
 // @g　室内に入った瞬間、かちん、と撃鉄の落ちる音を聞いた。"撃鉄" is a gun's hammer, not a regular hammer.
-@gThe moment I enter the room, I hear a 'kaching', the sound of a hammer striking.
+@gThe moment I enter the room, I hear a %{i}kaching%{/i}, the sound of a hammer striking.
```

Comment:
The sound of a cocking hammer is usually written in text as `ch-ch`, that or I think `click`, `clink` or `ch-chkt` are good options too.

Reply:
Click works for me